modules/sizing-requirements-for-gitops.adoc

```diff
@@ -28,3 +28,47 @@ Optionally, you can also use the ArgoCD custom resource with the `oc` command to
 ----
```

Comment:
This is a comment about the table in the "" section. Based on IBM unit guide, we should refer units differently: https://www.ibm.com/docs/en/ibm-style?topic=measurement-units#defining-abbreviations
I would suggest making the change to 1024 MiB instead of 1024M, etc.

Reply:
This comment will be addressed by the following PR:

https://github.com/openshift/openshift-docs/pull/78989

Hence, I am marking this comment as resolved.